File: Modul-6/PRAK604/PRAK604-2310817210010-Harry-Pratama-Yunus.py
kalimat1=input()
kalimat2=input()
panjang1=len(kalimat1)
panjang2=len(kalimat2)
a=0
b=0
if panjang1 == panjang2:
    for i in range (panjang1):
        if kalimat1[i]==' ' and kalimat2[i]==' ':
            print(" ", end='')
            continue
        if kalimat1[i]==kalimat2[i]:
            print("*", end='')
            a=a+1
        else:
            print("#", end='')
            b=b+1
        if i==panjang1-1:
            print()
    print("*={}".format(a))
    print("#={}".format(b))
    if a>=b:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")
else:
    print("Panjang kalimat berbeda, pesan palsu")

File: Modul-6/PRAK604/PRAK604-2310817210010-Harry-Pratama-Yunus.c
#include<stdio.h>
#include<string.h>

int main(){
    char kalimat1[TMP_MAX],kalimat2[TMP_MAX];
    fgets(kalimat1,sizeof(kalimat1),stdin);
    fgets(kalimat2,sizeof(kalimat2),stdin);
    int panjang1=strlen(kalimat1),panjang2=strlen(kalimat2);
    int a=0,b=0;
    if (panjang1 == panjang2){
        for(int i=0;i<=panjang1-2;i++){
            if(kalimat1[i]==' '&&kalimat2[i]==' '){
                printf(" ");
                continue;
            }
            if(kalimat1[i]==kalimat2[i]){
                printf("*");
                a+=1;
            }
            else{
                printf("#");
                b+=1;
            }
            if(i==panjang1-2){
                printf("\n");
            }
        }
        printf("*=%d\n",a);
        printf("#=%d\n",b);
        if(a>=b){
            printf("Pesan Asli");
        }  
        else{
            printf("Pesan Palsu");
        }
    } 
    else{
        printf("Panjang kalimat berbeda, pesan palsu");
    }
    return 0;
}